copy/multiple.go

```diff
@@ -39,8 +42,15 @@ func prepareInstanceCopies(instanceDigests []digest.Digest, options *Options) []
 			logrus.Debugf("Skipping instance %s (%d/%d)", instanceDigest, i+1, len(instanceDigests))
 			continue
 		}
+		operation := instanceCopyCopy
+		forceZstd := false
+		if _, ok := options.EnsureCompressionVariantExists["zstd"]; ok {
```

Comment:
Some code somewhere, maybe here, needs to reject requests for any formats that we don’t currently support.

---

We _also_ should fail if this is set while copying a non-index image; or, eventually, implement that. (Notably that matters for `podman build && podman push`.